CMakeLists.txt

```diff
@@ -21,50 +18,51 @@ endif()
 #####################################
 # Define Options
 #####################################
-option(BUILD_EXAMPLES "Builds the examples in the project" ON)
-option(BUILD_TESTING  "Builds the tests in the project"    ON)
+option(CROW_BUILD_EXAMPLES "Build the examples in the project" ON )
+option(CROW_BUILD_TESTS    "Build the tests in the project"    ON )
+option(CROW_AMALGAMATE     "Combine all headers into one"      OFF)
 
 #####################################
-# Define CMake Module Imports
+# Define Targets
 #####################################
-include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/dependencies.cmake)
-include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/compiler_options.cmake)
+add_library(crow INTERFACE)
+target_include_directories(crow INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/include)
 
-#####################################
-# Define project-wide imports
-#####################################
-# this can be alternatively (and as recommended way) done with target_include_directories()
-if(BUILD_EXAMPLES OR BUILD_TESTING)
-	set(PROJECT_INCLUDE_DIR
-		${CMAKE_CURRENT_SOURCE_DIR}/include
-	)
```

Comment:
I'm guessing this is what's causing the CI build to fail

Reply:
I quickly rebased the repo yesterday but didn't link the new examples to the crow target which is leading to this error. Will be fixed.